docs/fundamentals/debugging/actors.md

```diff
@@ -121,3 +141,179 @@ If you are establishing a link definition but still see errors that indicate tha
 Previous guides used `wash ctl link query`, which is now deprecated and will be removed in a future version.
 See [the wash command refactoring RFC](https://github.com/wasmCloud/wash/issues/538) for more information and to provide feedback
 :::
+
+## Debugging Actor/Provider communication
+
+### `wash spy`
+
+As of wash 0.18, there is a new experimental debugging command called `wash spy`. This command
+allows you to see all messages sent between actors and providers, and is useful for debugging
+communication issues. To use it, you'll need to enable experimental mode for wash:
+
+```shell
+export WASH_EXPERIMENTAL=true
+```
+
+Then, you can run `wash spy` to see all messages sent between actors and providers. 
+
+```shell
+$ wash spy <actor name or ID>
+```
+
+You can pass the actual actor ID if you wish, but for ease of use, if a non-key is provided, wash
```

Comment:
> non-key

I would just drop the "if a non-key is provided" phrase and say "wash will attempt to resolve the argument to..."